database/crudOperations.js

```diff
@@ -0,0 +1,98 @@
+//need to import models into this file
+const {summaries, hosts} = require('./postgresSchema.js');
+
+const insertNewSummaryInfo = async (summaryInfo, cb) => {
+  //should recieve json
+  //summaryBody should include everything except hostName
+  let summaryBody = {...summaryInfo};
+  delete summaryBody.hostName;
+  //hostBody should include only hostName
+  let hostBody = {};
+  hostBody.hostName = summaryInfo.hostName;
```

Comment:
Feel like there might be a way to do this in less lines, maybe destructure hostname out of summaryInfo and assign to hostbody.. and then remove that property.  Could also be less taxing then creating two net new objects

Reply:
I thought so as well but couldn't think of any so I moved on. I will give it another go because there for sure is an easier way to do it.